build_defs/VERSION

```diff
@@ -0,0 +1 @@
+0.7.2
```

Comment:
Ideally we would have this `VERSION` file in the root of the repo, but a circular dependency is formed.